### ISSUE TITLE
How to make it working with requirejs

### ISSUE THREAD

**Issue by standino** [jonmiles/react-bootstrap-treeview#5]
When I use this with requirejs, I got error:

Uncaught ReferenceError: React is not defined react-bootstrap-treeview.js:1

Below is my step. please take a look and help. Thanks.

bower install react-bootstrap-treeview --save

* main.js

require.config({
	  deps: ["main"],

	  paths: {
		    react: '../bower_components/react/react-with-addons',
		    jsx: '../bower_components/jsx-requirejs-plugin/js/jsx',
		    JSXTransformer: '../bower_components/react/JSXTransformer',
		    jquery: '../bower_components/jquery/dist/jquery.min',
		    underscore: '../bower_components/underscore/underscore-min',
		    text: '../bower_components/requirejs-text/text',
		    backbone: '../bower_components/backbone/backbone',
               "react-bootstrap-treeview":'../bower_components/react-bootstrap-treeview/public/js/react-bootstrap-treeview'
	  },

* app.jsx

/** @jsx React.DOM */
define([
    'react',
    ...
    'react-bootstrap-treeview',
],function (React,...,TreeView){
...

  return (
               
                                <TreeView  defaultData={defaultData} />
               
            );

        }

**Comment by jonmiles** [comment]
I don't think it supports require.js out-of-the-box right now.  I've added #6 which should solve the problem in the future.

The meantime to get it working have you tried shim'in the resource in your require config.
